pim/utils.py

```diff
@@ -149,10 +150,10 @@ class spinner(threading.Thread):
     """
     def __init__(self):
         if os.name == 'posix':
-            self.chars = (unicodedata.lookup('FIGURE DASH'),u'\\ ',u'| ',u'/ ')
```

Comment:
Haha alright, so it would seem that this is a case of blind-leading-blind...  I would hazard a guess that the posix thing could be dropped and the whole if/else block replaced with just `self.chars = ('-','\\ ','| ','/ ')`. Thoughts?